### PR TITLE
Fix/update on custom order

### PR DESCRIPTION
**Pull request by alexxcheung**
Old code will be deleted after passing the test

### CODE REVIEW

File: LetsDinnerV2 MessagesExtension/Controller/EventDescriptionVC/EventDescriptionViewController.swift [hunk 1]
```diff
@@ -54,7 +54,7 @@ class EventDescriptionViewController: UIViewController {
     }
     
     private func mergeRecipesTitles() {
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/EventDescriptionVC/EventDescriptionViewController.swift]: mergeRecipeTitles

File: LetsDinnerV2 MessagesExtension/Controller/EventDescriptionVC/EventDescriptionViewController.swift [hunk 2]
```diff
@@ -54,7 +54,7 @@ class EventDescriptionViewController: UIViewController {
...
-        allRecipesTitles = Event.shared.mergeAllRecipesTitles(selectedRecipes: selectedRecipes, selectedCustomRecipes: selectedCustomRecipes)
+        allRecipesTitles = CustomOrderHelper.shared.mergeAllRecipesTitleInCustomOrder()
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Controller/EventDescriptionVC/EventDescriptionViewController.swift]: mergeAllRecipeTitlesInCustomOrder()

File: LetsDinnerV2 MessagesExtension/Model/Event.swift [hunk 3]
```diff
@@ -177,7 +177,8 @@ class Event {
                 
                   let parameters : [String : Any] = ["title" : recipe.title ?? "",
                                                      "sourceUrl" : recipe.sourceUrl ?? "",
-                                                    "customOrder":recipe.customOrder]
+                                                     "id" : recipe.id ?? 0,
```
Comment (Ericordo) [inline LetsDinnerV2 MessagesExtension/Model/Event.swift:180]: Default value should be a unique number, UUID.uuidString()

**Review by Ericordo — APPROVED** [review]

File: LetsDinnerV2 MessagesExtension/Model/Event.swift (hunk 3 above)
Comment (alexxcheung) [reply LetsDinnerV2 MessagesExtension/Model/Event.swift:180]: I think the default value is not necessary right? It is not useful even we have an unique ID. 
Its just for solving the optional type. How about guard let the recipe.id?

Comment (Ericordo) [reply LetsDinnerV2 MessagesExtension/Model/Event.swift:180]: But then if you do guard let and it has no id it will return without completing the function

Comment (alexxcheung) [reply LetsDinnerV2 MessagesExtension/Model/Event.swift:180]: The id is for matching with the id which saved in customOrder. 
If the id is a default value, they wont be matched. That means the id is not useful if it is a default value. 
So, it doesn't matter what the default value is.

Comment (Ericordo) [reply LetsDinnerV2 MessagesExtension/Model/Event.swift:180]: Alright

**Pull request merged**